lib.php

```diff
@@ -1251,6 +1251,14 @@ function facetoface_get_userfields() {
                 $userfields[$shortname] = get_string($shortname);
             }
             $userfields['managersemail'] = get_string('manageremail', 'facetoface');
+
+            if ($DB->record_exists('user_info_field', array('shortname' => 'stream'))) {
+                $userfields['stream'] = get_string('stream', 'facetoface');
+            }
+
+            if ($DB->record_exists('user_info_field', array('shortname' => 'division'))) {
```

Comment:
It won't matter since we'll need to use a different method anyway, but in this case the `shortname` for the Division field is actually 'level7' so the column isn't being added to the export.
It's always a good idea to double check how it's defined in _Site administration > Users > User profile fields_.